content/en/developers/metrics/metrics_units.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Metrics Units
+kind: documentation
+further_reading:
+- link: "developers/dogstatsd"
+  tag: "Documentation"
+  text: "Learn more about DogStatsD"
+- link: "developers/libraries"
+  tag: "Documentation"
+  text: "Official and Community-contributed API and DogStatsD client libraries"
+---
+
+To eliminate ambiguity and help you make sense of your systems, the following units may be associated with metrics submitted to Datadog.
```

Comment:
Are the `units` just examples of each `type`, or are those actual, specific implementations in our platform? (i.e. could I use `kelvin`?)

Reply:
those are all supported type within the app

by supported it means (autoscale in K, M, G)